packages/@webex/plugin-meetings/src/interpretation/index.ts

```diff
@@ -121,14 +121,31 @@ const SimultaneousInterpretation = WebexPlugin.extend({
    * Update self's interpretation information (self is interpreter)
    * @param {Object} interpretation
    * @param {String} selfParticipantId
-   * @returns {void}
+   * @returns {bool} is target language changed
    */
   updateSelfInterpretation({interpretation, selfParticipantId}) {
+    this.set('selfIsInterpreter', true);
+    const preTargetLanguage = this.targetLanguage;
     const {originalLanguage, sourceLanguage, order, isActive, targetLanguage, receiveLanguage} =
       interpretation || {};
     this.set({originalLanguage, sourceLanguage, order, isActive, targetLanguage, receiveLanguage});
     this.set('selfParticipantId', selfParticipantId);
+
+    return !!(targetLanguage && preTargetLanguage !== targetLanguage);
   },
+
+  /**
+   * Get the language code of the interpreter target language
+   * @returns {number}
+   */
+  getTargetLanguageCode() {
+    if (this.selfIsInterpreter) {
+      return this.siLanguages.get(this.targetLanguage)?.languageCode;
```

Comment:
should getTargetLanguageCode() take into account also the value of `this.isActive`?

Reply:
I don't think getTargetLanguageCode() should take into account also the value of this.isActive, because targeLanguage is always there no matter is active or not. But you reminded me that we need take into account the value when call setSendNamedMediaGroups()!